doc/frequency_evaluation_framework.md

```diff
@@ -0,0 +1,373 @@
+   </td>
+  </tr>
+  <tr>
+   <td>M
+   </td>
+   <td>Implement <code><a href="https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/src/simulations/set_generator.py">SetGenerator</a></code> for Scenario 3: Publisher Constant Frequency.
+   </td>
+  </tr>
+  <tr>
+   <td>3 x L
+   </td>
+   <td>Create <code>StratifiedSketch.</code>
+   </td>
+  </tr>
+  <tr>
+   <td>M
+   </td>
+   <td>Modify <code><a href="https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/src/simulations/simulator.py">simulator</a></code> to rename columns in output DataFrame and to add additional columns for higher frequencies.
+   </td>
+  </tr>
+  <tr>
+   <td>M
+   </td>
+   <td>Modify <code><a href="https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/src/simulations/simulator.py">simulator</a></code> to compute shuffle distance metric and include it in the output DataFrame.
+   </td>
+  </tr>
+  <tr>
+   <td>S
+   </td>
+   <td>Modify <code><a href="https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/src/evaluations/configs.py">ScenarioConfig</a></code> to include <code>max_frequency</code>
+   </td>
+  </tr>
+  <tr>
+   <td>L
+   </td>
+   <td>Create <code>FrequencyEstimatorEvaluationAnalyzer</code> class similar to <code><a href="https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/src/evaluations/analyzer.py">CardinalityEstimatorEvaluationAnalyzer</a></code>.  Create functions to calculate number of sketches that can be unioned and generate plots of true and estimated frequency distributions.
+   </td>
+  </tr>
+  <tr>
+   <td>L
+   </td>
+   <td>Create a <code>FrequencyReportGenerator</code> class for generating reports from frequency comparisons, similar to existing <code>ReportGenerator</code>.
+   </td>
+  </tr>
+  <tr>
+   <td>S
+   </td>
+   <td>Modify <code><a href="https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/src/evaluations/evaluator.py">run_one_scenario</a></code> so that it passes <code>max_frequency</code> to the Simulator.  Modify to use <code>FrequencyEstimatorEvaluationAnalyzer</code> and <code>FrequencyReportGenerator</code> if <code>max_frequency</code> > 1.
+   </td>
+  </tr>
+  <tr>
+   <td>M
+   </td>
+   <td>Implement sample config.
+   </td>
+  </tr>
+</table>
+
+### Links
+
+#### Docs
+
+[[Draft for WFA] Cardinality and Frequency Estimation Evaluation Framework](https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/doc/cardinality_and_frequency_estimation_evaluation_framework.md)
```

Comment:
```suggestion
[Cardinality and Frequency Estimation Evaluation Framework](https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/doc/cardinality_and_frequency_estimation_evaluation_framework.md)
```
Suggest dropping the draft because it's already published.